### ISSUE TITLE
Input type about calculating mean_iou

### ISSUE THREAD

**Issue by Wolfwjs** [open-mmlab/mmsegmentation#1614]
When mmseg calculates `iou`, do  `pred_label` and `label` need to be of type `onehot`? What exactly is their size？
<img width="673" alt="image" src="https://user-images.githubusercontent.com/36124147/170398736-70772b64-a387-4c26-a42d-55fcea6f2501.png">

**Comment by linfangjian01** [comment]
yep，(H, W) (i.e. a seg_logits and a label)
